CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 
 ### Fixed
 
+[#4471](https://github.com/thanos-io/thanos/pull/4471) **breaking:** Query/Rule/Compact/Store: Serve `/debug/*`, `/-/ready`, `/-/healthy`, and `/metrics` routes under the prefix from `--web.route-prefix`, if set.
```

Comment:
Just to clarify it, let's start with `**breaking** if --web.route-prefix is set: serves endpoints x,y,z now actually under the route-prefix.

Reply:
✅